rfc/rfc021-vp_token-grant-type.md

```diff
@@ -148,7 +148,8 @@ In addition to the error response defined in OAuth 2.0 [RFC6749], the Authorizat
 
 * `invalid_verifiable_presentation`: The VP is invalid. This error code is used when the signature is incorrect or when a required field is missing.
 * `invalid_presentation_submission`: The Presentation Submission is invalid. This error code is used when the Presentation Submission is not an answer to the Presentation Definition that corresponds with the requested scope.
-* `invalid_verifiable_credentials`: The submitted Verifiable Credentials do not meet the requirements. This error code is used when the Verifiable Credentials aren't corresponding to the Presentation Definition or when the Verifiable Credentials are expired, not trusted or invalid. 
+
+An `invalid_request` is returned for any submitted Verifiable Credentials that do not meet the requirements, when the Verifiable Credentials aren't corresponding to the Presentation Definition or when the Verifiable Credentials are expired, not trusted or invalid. 
```

Comment:
I wonder whether these error codes are too detailed, since the client won't act any differently depending on which of these 3 is returned. It also requires the implementation to map to these exact errors (which might be cumbersome). These 3 could also be "invalid_request" witth additional information in the "description" field?

Reply:
There could be a difference between a technical error and user error. Technical would be during development and user error would mean the user loaded the incorrect VCs in the wallet?